markdown/developers/guidelines/requirements/workflow_specificity.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Workflow specificity
+subtitle: There should only be a single pipeline per data / analysis type.
+menu:
+  main:
+    weight: 20
+---
+
+The nf-core community was founded to allow different groups to collaborate on pipelines instead of reinventing the same workflows in each institute.
```

Comment:
Always ask for advice on #new-pipelines?

Reply:
Noted down the same thing a bit more down in another section too :-)

Reply:
Added 👍🏻